app/src/main/java/br/com/concrete/magic/domain/model/Expansion.kt

```diff
@@ -8,8 +8,15 @@ data class Expansion(
     var code: String,
     var name: String,
     var type: String,
-    var booster: List<String>,
+//    var booster: List<String>,
     var releaseDate: String,
-    var block: String,
+    var block: String? = null,
     var onlineOnly: Boolean
-)
\ No newline at end of file
+) : Comparable<Expansion> {
+    override fun compareTo(other: Expansion): Int {
+        if (this.name > other.name) return 1
+        if (this.name < other.name) return -1
+        return 0
+    }
```

Comment:
Uma outra forma de escrever esse método fazendo uso do kotlin seria:
```suggestion
    override fun compareTo(other: Expansion) = when {
        this.name > other.name -> 1
        this.name < other.name -> -1
        else -> 0
    }
```